### DISCUSSION TITLE
Example of sites using document caching

### DISCUSSION THREAD

**Discussion by Biboswan** [urql-graphql/urql#2791] [Q&A] [answered]
I'm doing a case study between the two cache types and would appreciate some sample sites where document caching is being used. There must have been research behind choosing a simpler cache over graphcache as the default. Also, when can document caching be more efficient than normalised caching

**Comment by JoviDeCroock** [comment, marked as answer]
I personally don't know who's using document caching and who's using normalized caching, however I have been on a few projects that have transitioned from document to normalized caching as their application was growing. Full disclaimer I am not entirely sure all of these needed to transition but my personal feeling towards it has always been that if you have a lot of queries that could go stale due to mutations that more often than not you might need to consider normalized caching.

The benefits of document caching for me are the easy invalidation leading to refetches and having to think less about keys and how data is shared. The benefits of normalized caching are pretty big in terms of performance/... you can bypass a lot of refetching, share data across queries, ... but the trade-off ofcourse is the size aspect.
